File: poo-php/src/Outil.php
<?php

class Outil
{
    private string $nom;
    private float $prixParJour;
    private bool $disponible;

    public function __construct(string $nom, float $prixParJour, bool $disponible = true)
    {
        $this->nom = $nom;
        $this->prixParJour = $prixParJour;
        $this->disponible = $disponible;
    }

    public function getNom(): string
    {
        return $this->nom;
    }

    public function getPrixParJour(): float
    {
        return $this->prixParJour;
    }

    public function estDisponible(): bool
    {
        return $this->disponible;
    }

    public function setDisponible(bool $disponible): void
    {
        $this->disponible = $disponible;
    }

    public function __toString(): string
    {
        $status = $this->disponible ? "DISPONIBLE" : "LOUE";
        return $this->nom . " | " . $this->prixParJour . " €/jour | " . $status;
    }
}

File: poo-java/src/Outil.java
public class Outil {
    private String nom;
    private double prixParJour;
    private boolean disponible;

    public Outil(String nom, double prixParJour, boolean disponible) {
        this.nom = nom;
        this.prixParJour = prixParJour;
        this.disponible = disponible;
    }

    public String getNom() {
        return nom;
    }

    public double getPrixParJour() {
        return prixParJour;
    }

    public boolean isDisponible() {
        return disponible;
    }

    public void setDisponible(boolean disponible) {
        this.disponible = disponible;
    }

    @Override
    public String toString() {
        return nom + " | " + prixParJour + " €/jour | " + (disponible ? "DISPONIBLE" : "LOUE");
    }
}
